From 814421a53454a9d35cd9d0799f68eaae8b7cf440 Mon Sep 17 00:00:00 2001
From: Roger Zanoni <rzanoni@igalia.com>
Date: Fri, 7 Apr 2023 16:28:51 +0200
Subject: [PATCH 20/33] [M118-fix] Don't look for depot_tools in chrommium's
 third_party folder

This caused an issue at the configure step before we starte using
tarballs, we need to check if it still needed.

Upstream-Status: Inappropriate
Signed-off-by: Roger Zanoni <rzanoni@igalia.com>
---
 build/find_depot_tools.py | 8 --------
 1 file changed, 8 deletions(-)

diff --git a/build/find_depot_tools.py b/build/find_depot_tools.py
index f891a414837a5..592bba699c00d 100755
--- a/build/find_depot_tools.py
+++ b/build/find_depot_tools.py
@@ -27,14 +27,6 @@ def IsRealDepotTools(path):
 
 def add_depot_tools_to_path():
   """Search for depot_tools and add it to sys.path."""
-  # First, check if we have a DEPS'd in "depot_tools".
-  deps_depot_tools = os.path.join(SRC, 'third_party', 'depot_tools')
-  if IsRealDepotTools(deps_depot_tools):
-    # Put the pinned version at the start of the sys.path, in case there
-    # are other non-pinned versions already on the sys.path.
-    sys.path.insert(0, deps_depot_tools)
-    return deps_depot_tools
-
   # Then look if depot_tools is already in PYTHONPATH.
   for i in sys.path:
     if i.rstrip(os.sep).endswith('depot_tools') and IsRealDepotTools(i):
-- 
2.42.1

